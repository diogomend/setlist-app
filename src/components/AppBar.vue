<template>
  <div class="app-bar-wrapper">
    <v-app-bar fixed dark>
      <v-app-bar-nav-icon @click="toggleDrawer(true)"></v-app-bar-nav-icon>
      <v-btn v-if="isMobile" @click="$router.go(-1)" icon>
        <v-icon>arrow_back</v-icon>
      </v-btn>
      <v-toolbar-title>Diogo Mendon√ßa</v-toolbar-title>
    </v-app-bar>
  </div>
</template>

<script>
import { mapActions } from "vuex";
import mobileMixin from "@/mixins/mobileMixin";
export default {
  name: "AppBar",
  mixins: [mobileMixin],
  methods: {
    ...mapActions("appState", ["toggleDrawer"])
  }
};
</script>
