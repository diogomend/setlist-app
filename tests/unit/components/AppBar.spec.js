import Vue from "vue";
import { shallowMount } from "@vue/test-utils";
import AppBar from "@/components/AppBar";
import Vuetify from "vuetify";

Vue.use(Vuetify);

describe("AppBar.vue", () => {
  it("is a Vue instance", () => {
    const wrapper = shallowMount(AppBar);

    const template = wrapper.html();
    expect(template).toMatchSnapshot();
  });
});
