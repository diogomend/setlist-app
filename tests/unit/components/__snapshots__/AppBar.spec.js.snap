// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppBar.vue is a Vue instance 1`] = `
<div class="app-bar-wrapper">
  <v-app-bar-stub dark="true" tag="header" tile="true" extensionheight="48" src="" value="true" fixed="true">
    <v-app-bar-nav-icon-stub></v-app-bar-nav-icon-stub>
    <!---->
    <v-toolbar-title-stub>Diogo Mendon√ßa</v-toolbar-title-stub>
  </v-app-bar-stub>
</div>
`;
